WeScanSampleProject/ImplementEditImageViewController.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  ImplementEditImageViewController.swift
+//  WeScanSampleProject
+//
+//  Created by Chawatvish Worrapoj on 8/1/2563 BE.
+//  Copyright © 2563 WeTransfer. All rights reserved.
+//
+
+import UIKit
+import WeScan
+
+class ImplementEditImageViewController: UIViewController {
```

Comment:
We can make this class `final`. Also, it's not best practice to call classes with words like "Implement"

```suggestion
final class EditImageViewController: UIViewController {
```

Reply:
Got it. 🙏🏼